src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -35,7 +35,7 @@
     <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
     <_FullFrameworkReferenceAssemblyPaths>$(MSBuildThisFileDirectory)/Documentation</_FullFrameworkReferenceAssemblyPaths>
     <SkipCommonResourcesIncludes>true</SkipCommonResourcesIncludes>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
I was wondering if it might be better to just set this in the `Directory.Build.props` in the repo root (so it can also cover test projects, etc)....

It might also be worthwhile to note that we should be able to set this to `preview` in the next update: https://github.com/dotnet/roslyn/pull/33089

Reply:
Moved it to be centrally managed in `dir.common.props`